tools/bin/build_status.py

```diff
@@ -0,0 +1,69 @@
+#!/usr/bin/env python3
+
+from collections import defaultdict
+import requests
+import json
+
+"""
+Create tools/bin/secrets/build_status.json with the secret in the "build stat secrets" in LastPass
+"""
+
+with open('tools/bin/secrets/build_status.json') as f:
+    secrets = json.load(f)
+
+res = requests.get(f"https://kvdb.io/{secrets['bucket']}/?values=true&format=json",
+                   auth=(secrets['secret_key'], '')).json()
+
+connector_to_states = defaultdict(list)
+
+for stat in res:
+    connector = stat[0][0:stat[0].rfind('-')]
+    run_id = stat[0][stat[0].rfind('-') + 1:len(stat[0])]
+    status = stat[1][0:stat[1].rfind('-')]
+    timestamp = stat[1][stat[1].rfind('-') + 1:len(stat[1])]
+    connector_to_states[connector].append([timestamp, run_id, status])
+
+print("-" * 81)
```

Comment:
Just a suggestion to help read the table, was not obvious looking at the table which side is the "important" one
```suggestion
print("Most recent <----- Least Recent")
print("-" * 81)
```